### DISCUSSION TITLE
Option to trigger readings on an Device (IDX)

### DISCUSSION THREAD

**Discussion by Micha-123** [addiejanssen/domoticz-solaredge-modbustcp-plugin#66] [Ideas]
Is it possible to add an option to trigger the readings with an IDX? Not only every some seconds? Simply an chose time based (and seconds) like now or switching the trigger to an device, if an specific IDX is updated (triggered) then the plug-in reada the dajta from SolarEdge inverter.

The reason is, i need to trigger it on my P1 meter, then it reads the data from SolarEdge, and my dzvents script calculates the real consumption of power (Solar Edge + Grid)

**Comment by addiejanssen** [comment]
Hi @Micha-123,

I have no idea....
Can you provide (a link to) a sample/other plugin that provides such an option?
